Dockerfile.compute-node

```diff
@@ -241,9 +241,74 @@ RUN wget https://github.com/df7cb/postgresql-unit/archive/refs/tags/7.7.tar.gz -
 FROM build-deps AS vector-pg-build
 COPY --from=pg-build /usr/local/pgsql/ /usr/local/pgsql/
 
+# Kaniko doesn't support heredoc in dockerfiles[0], so we echo the whole patch (with \n and escaping).
```

Comment:
Maybe it's better to create a folder and just this file?

Reply:
Yep, that makes sense, moved the patch to a directory